Documentation/community/roadmap.rst

```diff
@@ -14,35 +14,60 @@ broader community. You'll also find here some pointers on how you can
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gsg_encryption` (IPSec and WireGuard)     | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`bandwidth-manager`                        | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+| Cilium Mesh                                      | Stable (:ref:`Roadmap Details<rm-clustermesh>`)          |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Multi-Cluster (ClusterMesh)<clustermesh>` | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`External Workloads<external_workloads>`   | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+| Hubble Observability                             | Stable (:ref:`Roadmap Details<rm-hubble-observability>`) |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Hubble CLI<hubble_cli>`                   | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`hubble_ui`                                | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Prometheus metrics<metrics>`              | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+| Cilium Service Mesh                              | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)  |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_ingress`                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_gateway_api`                           | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_l7_traffic_management`                 | Stable                                                   |
```

Comment:
Is this really stable? 2/3 of the [examples](https://docs.cilium.io/en/v1.14/network/servicemesh/l7-traffic-management/#examples) are using CiliumEnvoyConfig which I don't think we make any stability guarantees about, and the other example which doesn't use CEC is explicitly marked beta.

Reply:
Yeah, I would have thought this should be a bit lower for now too.

Reply:
good catch, I meant to mark Gateway API as Beta so that was my mistake. Sounds like L7 traffic mgmt should be too, so I’ll change that as well